go/README.md

```diff
@@ -0,0 +1,19 @@
+# PiranhaGo
+PiranhaGo is runnable now. 
+Instructions:-
+1. To build the package run `go build -o piranha`. Dependencies will install automatically and they are given in `go.mod` file.
+2. Below are the instructions for running for the single file.
+```
+Inputs from args
+Usage: ./piranha [-h] -p PROPERTIES -s SOURCE_FILE -f STALE_FLAG [-treated] [-o OUTPUT]
+Required arguments:
+		-s SOURCE_FILE: Path of the file to be refactored
+		-p PROPERTIES: Configuration file (json format) for Piranha.
+		-f STALE_FLAG: Name of the stale flag
+Optional arguments:
+		-h: Show the options and exit.
+		-treated: If this is given, then flag is treated,
+			otherwise it is control.
```

Comment:
We can make it different. But I thought making it implicit is still fine. I think it's generally less pain to first-time users as they have to set fewer attributes initially when they are trying out. (I will answer other comments slowly and steadily:)

Reply:
My thinking here is that, since we are deleting code in place, we do want the user to be somewhat explicit about which logic ought to be deleted, even if it's a bit more friction. It's also not clear that one default would be more common than the other.

Reply:
Hmm, I agree with you. I suggest we can ask for as like:
`-mode treated` or `-mode control`. Is it fine?

Reply:
Sure. That makes sense to me :)